pages/agent/v3/help/_bootstrap.md

```diff
@@ -65,13 +65,14 @@ See https://buildkite.com/docs/agent/v3/hooks for more details.
 <tr id="git-clone-mirror-flags"><th><code>--git-clone-mirror-flags value</code> <a class="Docs__attribute__link" href="#git-clone-mirror-flags">#</a></th><td><p>Flags to pass to "git clone" command when mirroring (default: "-v")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CLONE_MIRROR_FLAGS</code></p></td></tr>
 <tr id="git-clean-flags"><th><code>--git-clean-flags value</code> <a class="Docs__attribute__link" href="#git-clean-flags">#</a></th><td><p>Flags to pass to "git clean" command (default: "-ffxdq")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CLEAN_FLAGS</code></p></td></tr>
 <tr id="git-fetch-flags"><th><code>--git-fetch-flags value</code> <a class="Docs__attribute__link" href="#git-fetch-flags">#</a></th><td><p>Flags to pass to "git fetch" command<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_FETCH_FLAGS</code></p></td></tr>
-<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands. For example, <code>update --init</code>. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
+<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands. For example, `update --init`. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
```

Comment:
What's the reason for this change? It looks like it shows the backticks rather than using the monospace code font.

Reply:
i'm not 100% sure why this changed, but i suspect it has to do with [prime signs](https://github.com/buildkite/agent/blob/main/clicommand/prime-signs.md?plain=1) thing, which has historically been a bit janky and doesn't work that great

Reply:
😆 I really appreciate that you documented that. TIL: prime signs. 

I assume this is another quirk of the doc generation. I see that other fields have the same issue in their descriptions, but the example values display correctly 🫠